hspec-core/src/Test/Hspec/Core/Tree.hs

```diff
@@ -135,6 +143,7 @@ specItem s e = Leaf Item {
   , itemIsParallelizable = Nothing
   , itemIsFocused = False
   , itemExample = safeEvaluateExample e
+  , itemMaxTime = Nothing
```

Comment:
Same here, I think this can be deferred to #891.

Reply:
I am unsure where to get the timeout duration from if this is removed.